### PR TITLE
feat(telekom-header): no shadow for `nav-list` and `nav-item`

### PR DESCRIPTION
**Pull request by acstll**
Ideally we merge #1477 first.

### CODE REVIEW

File: packages/components/src/components/telekom/telekom-nav-item/telekom-nav-item.tsx [hunk 1]
```diff
@@ -9,22 +9,56 @@
...
+      return;
+    }
+    toggleAriaCurrent(this.link, newValue);
+  }
+
+  connectedCallback() {
+    this.activeChanged(this.active);
+  }
+
+  get link(): HTMLAnchorElement | null {
```
Comment (nowseemee) [inline packages/components/src/components/telekom/telekom-nav-item/telekom-nav-item.tsx:53]: 💡

Comment (acstll) [reply packages/components/src/components/telekom/telekom-nav-item/telekom-nav-item.tsx:53]: is this good or bad? 😅

Comment (nowseemee) [reply packages/components/src/components/telekom/telekom-nav-item/telekom-nav-item.tsx:53]: it's very, very, very good, chef!

**Review by nowseemee — APPROVED** [review]
seems pretty good to me!

**Pull request merged**